examples/complete/main.tf

```diff
@@ -0,0 +1,13 @@
+module "yaml_config" {
+  source = "../../"
+
+  map_yaml_config_local_base_path = path.cwd
```

Comment:
Since the module is all about YAML, I think we can drop `_yaml_` from the variables so they are shorter.